team-validator.js

```diff
@@ -310,6 +311,14 @@ class Validator {
 						} else {
 							problemString = problemString.concat(`.`);
 						}
+						// Sketchmons hack part 2
+						if (allowOneSketch) {
+							if (noSketch.indexOf(move.name) >= 0) {
+								problemString = problemString.concat(` (${move.name} is banned from being sketched in ${format.name}.)`);
```

Comment:
Is this going to cause a problem even if the move is legally obtainable by other methods?

Reply:
If we get this far, it means the move isn't normally legal.

Reply:
Oh, I see now.